.gitignore

```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
+=======
```

Comment:
This shows a merge conflict, which needs to be resolved before the code can be pulled